apps/web/lib/getMainDomainOrgRedirect.ts

```diff
@@ -0,0 +1,45 @@
+import type { IncomingMessage } from "http";
+
+import { WEBAPP_URL } from "@calcom/lib/constants";
+import logger from "@calcom/lib/logger";
+
+const log = logger.getSubLogger({ prefix: ["lib", "getMainDomainOrgRedirect"] });
+export const getMainDomainOrgRedirect = (
+  req: IncomingMessage | undefined,
```

Comment:
As I mentioned earlier, I don't think this method needs entire req. It just needs the URL

Reply:
since we need host from req.headers.host , we need to pass req as param